inference-engine/src/mkldnn_plugin/nodes/mkldnn_pad_node.cpp

```diff
@@ -306,12 +306,10 @@ void MKLDNNPadNode::PadExecutor::exec(MKLDNNMemoryPtr& srcMemPtr, MKLDNNMemoryPt
 }
 
 void MKLDNNPadNode::execute(mkldnn::stream strm) {
-    auto& dstMemPtr = getChildEdgeAt(0)->getMemoryPtr();
-    auto& srcMemPtr = getParentEdgeAt(0)->getMemoryPtr();
-    if (!dstMemPtr || !srcMemPtr)
-        THROW_ERROR << "has not allocated source/destination memory.";
+    if (!execPtr)
+        THROW_ERROR << "has not compiled executor.";
```

Comment:
"doesn't have a compiled executor" or "executor has not been compiled"

Reply:
I used macro `#define THROW_ERROR IE_THROW() << "Pad layer with name '" << getName() << "' "` to avoid write it everywhere. So I select exception text for this macro